src/Rector/AbstractScopeAwareRector.php

```diff
@@ -24,19 +21,18 @@ abstract class AbstractScopeAwareRector extends AbstractRector implements ScopeA
      */
     public function refactor(Node $node): int|array|Node|null
     {
-        /** @var MutatingScope|null $currentScope */
-        $currentScope = $node->getAttribute(AttributeKey::SCOPE);
-
-        if (! $currentScope instanceof Scope) {
-            $errorMessage = sprintf(
-                'Scope not available on "%s" node, but is required by a refactorWithScope() method of "%s" rule. Fix scope refresh on changed nodes first',
-                $node::class,
-                static::class,
-            );
+        trigger_error(
```

Comment:
Just throw exception here :+1:

Reply:
ok, updated to just throw https://github.com/rectorphp/rector-src/pull/6541/commits/5f03843ae80b6635043e82456074a5465cdd8baf